source/renderer/dialog/preferences.js

```diff
@@ -125,11 +125,7 @@ class PreferencesDialog extends ZettlrDialog {
     $('.dicts-list-search').on('keyup', (e) => {
       let val = $('.dicts-list-search').val().toLowerCase()
       $('.dicts-list').find('li').each(function (i) {
-        if ($(this).text().toLowerCase().indexOf(val) === -1) {
-          $(this).hide()
-        } else {
-          $(this).show()
-        }
+        this.style.display = this.textContent.toLocaleLowerCase().includes(val) ? 'block' : 'none'
```

Comment:
This is an interesting case, because I am extremely sure we shouldn't do it this way, but rather with helper classes – this way we don't need to expect the original `display` to be set to `block`, as this would not necessarily work on inline-elements, so please revert that change, I'll see to including a corresponding helper class in my PR #1062

Reply:
I picked this approach, because my understanding is that this is exactly what jQuery does under the hood: https://github.com/jquery/jquery/blob/master/src/css/showHide.js#L22

I wanted to keep the implementation as close to it as possible to reduce risk and as slim as possible. Happy to use another approach if there is a helper available.